src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -41,6 +41,9 @@
      */
     public String phoneNumber;
 
+    /** The user's preferred language **/
+    public String preferredLanguage;
```

Comment:
Can you add an example in the comment?

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/164/commits/4d1e10ecaf3ed18039d2a49a9863cb156d0cc9eb

Reply:
Haha I was kinda expecting that! Then it's probably best to rename this field to `preferredLocale`.

Reply:
Hmm it was a trap.. I'll update right now ☺️